pkg/util/pem_test.go

```diff
@@ -49,9 +52,10 @@ func TestValidateAndSanitizePEMBundle(t *testing.T) {
 			parts:     []string{dummy.TestCertificate1, string(poisonComment), dummy.TestCertificate2, randomComment, dummy.TestCertificate3, string(poisonComment)},
 			expectErr: false,
 		},
-		"valid bundle with expired cert succeeds": {
-			parts:     []string{dummy.TestCertificate1, dummy.TestExpiredCertificate},
-			expectErr: false,
+		"valid bundle with expired cert succeeds with the expired cert intact": {
+			parts:              []string{dummy.TestCertificate1, dummy.TestExpiredCertificate},
+			expectExpiredCerts: true,
+			expectErr:          false,
```

Comment:
Could it make sense to omit fields with the correct value for the test case? I makes things a bit more compact.

Reply:
Could you rephrase that for me? I don't think my morning coffee is kicking in yet and I'm not sure what you mean!

Reply:
Sorry! I just meant that the zero value of a `bool` in Golang is `false`. When do we explicitly set at field to the zero value in the test, and when do we not do it? I think the most common is to omit setting the field.

Reply:
Ahhh, I'm with you! It's a fair point.

In tests I like to be explicit about them because that makes the test cases easier to copy+paste and change (although I've not been explicit here).

I should be consistent, but I'm gonna merge this since it won't break anything.